js/source/esri-vector-tile-source.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+var util = require('mapbox-gl/js/util/util');
```

Comment:
Why are these includes of the library name rather than relative paths like `'../js/util/util'`? The latter is the case with the rest of the library as well as node.js customs.